scripts/render_DESeq2_report.R

```diff
@@ -88,7 +88,7 @@ if (is.na(params$group_facet)) {
   message(paste0("Making multiple reports based on ",
                  params$group_facet ,"..."))
   print(facets)
-  for (i in facets[2:length(facets)]) {
+  for (i in facets) {
```

Comment:
Not sure what was going on here, I looked at the past commits and it was 6-length and then 2-length, but this is clearly better.

Reply:
Yup... that's definitely my fault. Sometimes if it crashes at one facet, rather than repeat them all, I'll just subset it. Sorry! Not sure how those snuck in. haha